src/server.rs

```diff
@@ -190,8 +191,8 @@ impl<State: Send + Sync + 'static> Server<State> {
     /// };
     ///
     /// // Initialize the application with state.
-    /// let mut app = tide::with_state(state);
-    /// app.at("/").get(|req: Request<State>| async move {
+    /// let mut app = tide::with_state(Arc::new(state));
```

Comment:
This State could also have a `derive(Clone)` and not need to change the request signature